results/pkg/watcher/reconciler/controller_test.go

```diff
@@ -57,6 +55,20 @@ func reconcileTaskRun(ctx context.Context, asset test.Assets, taskRun *v1beta1.T
 	return tr, nil
 }
 
+// reconcileTaskRun sends TaskRun data to reconciler and then retrieves completed TaskRun
+func reconcilePipelineRun(ctx context.Context, asset test.Assets, pipelineRun *v1beta1.PipelineRun) (*v1beta1.PipelineRun, error) {
+	c := asset.Controller
+	clients := asset.Clients
+	if err := c.Reconciler.Reconcile(ctx, strings.Join([]string{pipelineRun.Namespace, pipelineRun.Name}, "/")); err != nil {
```

Comment:
Discovered this trick recently: https://pkg.go.dev/github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1#PipelineRun.GetNamespacedName

```suggestion
	if err := c.Reconciler.Reconcile(ctx, pipelineRun.GetNamespacedName()); err != nil {
```

Reply:
Got it!